scripts/repo.js

```diff
@@ -0,0 +1,24 @@
+(function(module) {
+  var repos = {};
+
+  repos.all = [];
+
+  repos.requestRepos = function(cb) {
+    $.ajax({
+      url: 'https://api.github.com/users/pjosifovic/repos' + '?per_page=4&sort=updated',
+      type: 'GET',
+      headers: { 'Authorization': 'token ' + githubToken},
+      success: function(data, message, xhr) {
+        repos.all = data;
+      }
+    }).done(cb);
+  };
```

Comment:
Consider adding a `fail` method to let your users know when the github ajax call.